homeassistant/components/renson/__init__.py

```diff
@@ -20,6 +20,7 @@
 
 PLATFORMS = [
     Platform.SENSOR,
+    Platform.FAN,
```

Comment:
I think that the convention is to alphabetically order this.

Reply:
Changed order + created PR for documentation